.github/workflows/build_and_run.yml

```diff
@@ -77,3 +77,57 @@ jobs:
           #       print(\"Validation:\",res)"                                        \
           #     )" | grep Validation | cut -f 2 -d ":"` || exit 1
           # if [ ${res} == 0 ]; then exit 1; fi
+
+  build_linux_mlir:
+    name: Builds dpbench and runs mlir benchmarks
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python: ["3.9", "3.10"]
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout dpbench
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.python }}
+          activate-environment: dpbench-dev-mlir
+
+      - name: Conda info
+        shell: bash -l {0}
+        run: conda info
+
+      - name: Install dpbench dependencies
+        shell: bash -l {0}
+        run: |
+          conda install -c intel tbb dpcpp_linux-64
+          conda install numpy numba cython cmake ninja scikit-build pandas
+          conda install scipy scikit-learn pybind11 tomli
+          conda install -c pkgs/main libgcc-ng">=11.2.0" libstdcxx-ng">=11.2.0" libgomp">=11.2.0"
+          conda install -c dppy/label/dev -c intel -c main dpctl numba-mlir dpnp
+          pip install alembic
+          conda list
+
+      - name: Build dpbench
+        shell: bash -l {0}
+        run: |
+          CC=icx CXX=icpx python setup.py develop -- -Dpybind11_DIR=$(python -m pybind11 --cmakedir) -DDPCTL_MODULE_PATH=$(python -m dpctl --cmakedir)
+
+      - name: Validate dpbench
+        shell: bash -l {0}
+        run: |
+          export OCL_ICD_FILENAMES=libintelocl.so
+          export NUMBA_MLIR_GPU_RUNTIME=sycl
+
+          python -c "import dpbench; dpbench.run_benchmarks(repeat=2, implementations=[\"numba_mlir_n\",\"numba_mlir_p\",\"numba_mlir_k\"])" || exit 1
```

Comment:
Could you try running this code in main pipeline, just after general call

Reply:
They seems to work, actually, so let's leave it in this state